packages/transaction-controller/CHANGELOG.md

```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [28.1.1]
+
+### Uncategorized
+
+- fix: simulation should handle NFT mints ([#4217](https://github.com/MetaMask/core/pull/4217))
```

Comment:
We like to be clear about the changes that are made to the API or to the behavior of the package rather than repeating the name of the commit message. If you were to explain this to a consumer of the package, how would you do it? (Tip: you could try summarizing your PR description, keeping in mind that consumers don't know about `getTokenBalanceTransactions` or `getTokenBalanceChanges`, since they're private to the `simulation` file, so we might not want to figure out a way to describe it more generally)

Reply:
Done. Thank you!